bin/wp-cli.php

```diff
@@ -334,7 +369,7 @@ public function debug( $args ) {
 
 		$timestamp_start = microtime( true );
 
-		WP_CLI::log( "Indexing post {$post_id}" );
+		WP_CLI::log( sprintf( 'Indexing post %1$d', $post_id ) );
```

Comment:
Why the change? `sprintf()` is generally harder to read than interpolation, and it's slower. In cases where we don't need to reformat the string or we don't need placeholders (e.g. for i18n), I personally much prefer interpolation

Reply:
(comment applies throughout)